lib/rubocop/cli.rb

```diff
@@ -66,9 +66,20 @@ def handle_exiting_options
       puts RuboCop::Version.version(false) if @options[:version]
       puts RuboCop::Version.version(true) if @options[:verbose_version]
       print_available_cops if @options[:show_cops]
+      print_target_files if @options[:list_target_files]
       exit(0)
     end
 
+    def print_target_files
+      finder = TargetFinder.new(@config_store, @options)
```

Comment:
I think it's better to implement this functionality in `Runner#run`. It saves you from having to duplicate the code from `Runner#find_target_files` here. If `@options[:list_target_files]` is true, print the files, otherwise inspect them.